config.ini

```diff
@@ -86,6 +89,13 @@ model = gpt-3.5-turbo
 ; 	The maximum number of sentences returned by the LLM. Lower this value to reduce waffling
 max_response_sentences = 999
 
+; alternative_openai_api_base:
+; If you are using openai's services, leave this alone, otherwise you can change this variable to another base_api that uses openai's api
+; For example, if you have a local llm framework or online framework that allows you to use a different url to access openai api functions, you can enter the base_api url here 
+; Your alternative api_base must support openai's python streaming protocol.  Examples: http://127.0.0.1:5001/v1 for textgenwebui using the default openai extension, or http://localhost:8080/v1 using the default endpoint for Local.ai.  Do NOT use quotes in entering the endpoint.
+; Note that for some services, like textgenwebui, you must enable the openai extension and have the model you want to use preloaded before running mantella.  Services may vary, read their documentation.
+; Leave this value as none to use the normal openai chat gpt models.
+alternative_openai_api_base = none
```

Comment:
Instead of settings this to a string with value none, can't you simply set default value as `https://api.openai.com/v1`?

And if you do that maybe rename to simply be `openai_api_base`